releases/v1.2.8.toml

```diff
@@ -0,0 +1,37 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.7"
+
+pre_release = false
+
+preface = """\
+The eighth patch release for `containerd` 1.2 provides a series of bug fixes, many
+of them backported from the master branch to correct several known issues around
+manifest lists/indexes and pulling multi-arch, CVEs related to Golang/gRPC and libseccomp,
+fd leakage, and finally mount cleanup related to Cloud Foundry and rootless containers.
+
+### Notable Updates
+* Skip rootfs unmount when no mounts are provided. Fixed by [PR #3148](https://github.com/containerd/containerd/pull/3148) {cherry-picked as [PR #3402](https://github.com/containerd/containerd/pull/3402)}.
+* Close inherited socket file descriptor. Fixed in [PR #3359](https://github.com/containerd/containerd/pull/3359) {cherry-picked as [PR #3364](https://github.com/containerd/containerd/pull/3364)}.
+* Bump libseccomp-golang to v0.9.1. Fixed in [PR #3371](https://github.com/containerd/containerd/pull/3371) {cherry-picked as [PR #3376](https://github.com/containerd/containerd/pull/3376)}.
```

Comment:
This one was (for now) reverted in https://github.com/containerd/containerd/pull/3538